src/main/java/cz/cvut/kbss/termit/service/IdentifierResolver.java

```diff
@@ -108,8 +104,18 @@ public static String normalize(String value) {
         Objects.requireNonNull(value);
         final String normalized = value.toLowerCase().trim()
                                        .replaceAll("[\\s/\\\\]", Character.toString(REPLACEMENT_CHARACTER));
-        return Normalizer.normalize(normalized, Normalizer.Form.NFD).replaceAll("[^\\p{ASCII}]", "")
-                         .replaceAll("[(?&)]", "");
+        return normalized.replaceAll("[(?&)]", "");
+    }
+
+    /**
+     * Normalizes the specified value which includes operations performed by {@link #normalize(String)} + replacing
+     * non-ASCII characters with ASCII ones (e.g., 'č' with 'c').
```

Comment:
It is not replacing č with c any more, right?

Reply:
No, this one is. The original `normalize` is not.